worker/utils/environment.js

```diff
@@ -69,12 +82,82 @@ class EnvironmentClass {
     return atlasPassword;
   }
 
-  static getFastlyServiceId() {
-    if (fastlyServiceId === undefined) {
+  static getFastlyMainServiceId() {
+    return fastlyMainServiceId;
+  }
+
+  static getFastlyMainToken() {
+    return fastlyMainToken;
+  }
+
+
+  static getFastlyDochubServiceId() {
+    if (fastlyDochubServiceId === undefined) {
       return 'testId';
     }
```

Comment:
I feel like this should probably log an error if it's important, or otherwise should just be removed - idk how much utility 'testId' provides here